src/tools/interfaces.md

```diff
@@ -14,7 +14,7 @@ The [Mozilla Growth & Usage Dashboard](https://gud.telemetry.mozilla.org/) (GUD)
 
 ### Glean Aggregated Metrics Dashboard (GLAM)
 
-The [Glean Aggregated Metrics Dashboard](https://glam.telemetry.mozilla.org/) (GLAM) is an interactive dashboard that is Mozilla’s primary self-service tool for examining the distributions of values of specific individual telemetry metrics, over time and across different user populations. It is similar to GUD in that it is meant to be usable by everyone; no specific data analysis or coding skills are needed. But while GUD is focused on a relatively small number of high level, derived product metrics about user engagement (e.g. MAU, DAU, retention, etc) GLAM is focused on a diverse and plentiful set of probes and data points that engineers capture in code and transmit back from Firefox and other Mozilla products.
+The [Glean Aggregated Metrics Dashboard](https://glam.telemetry.mozilla.org/) (GLAM) is an interactive dashboard that is Mozilla’s primary self-service tool for examining the distributions of values of specific individual telemetry metrics, over time and across different user populations.
```

Comment:
I'd probably keep the comparison with GUD on this page, as I think it might be helpful to people. Definitely link out to the more in-depth tutorial too.